Rebus.AdoNet.Tests/SagaPersisterTests.cs

```diff
@@ -278,7 +359,7 @@ public void WhenIgnoringNullProperties_DoesNotSaveNullPropertiesOnUpdate()
 		[Test]
 		public void WhenIgnoringNullProperties_DoesNotSaveNullPropertiesOnInsert()
 		{
-			var persister = CreatePersister(createTables: true);
+			var persister = _persister;
```

Comment:
let's get rid of this.. use _persister directly.

Reply:
ack